mpp/include/mpp_write_2Ddecomp.h

```diff
@@ -130,20 +130,16 @@
 !write time information if new time
           if( newtime )then
               if( KIND(time).EQ.DOUBLE_KIND )then
-                  error = NF_PUT_VAR1_DOUBLE( mpp_file(unit)%ncid, mpp_file(unit)%id, mpp_file(unit)%time_level, time )
+                  error = NF_PUT_VAR1_DOUBLE( mpp_file(unit)%ncid, mpp_file(unit)%id, mpp_file(unit:unit)%time_level, time )
```

Comment:
Why was this call changed to use `mpp_file(unit:unit)%time_level` as opposed to just `mpp_file(unit)%time_level` ?

Reply:
The interface checking in F90 throws an error about array/scalar mismatch.  The compiler interprets mpp_file(unit) as the array mpp_file(unit:).  By restricting the length of mpp_file with (unit:unit) to a single entry, the interface checking now sees mpp_file(unit:unit)%time_level properly.